Should sfv have a minimum-supported Rust version (MSRV) policy?

Currently it does not.

The first rust version that builds this crate successfully is 1.77

For 1.76 I get:
```bash
$ rustup run 1.76 cargo check
...
Checking sfv v0.10.4 (/home/icecold/workspace/sfv)
error[E0658]: use of unstable library feature 'round_ties_even'
   --> src/decimal.rs:167:46
    |
167 |         match Integer::try_from((v * 1000.0).round_ties_even() as i64) {
    |                                              ^^^^^^^^^^^^^^^
    |
    = note: see issue #96710 <https://github.com/rust-lang/rust/issues/96710> for more information

```

--- a/.github/workflows/ci-workflow.yml
+++ b/.github/workflows/ci-workflow.yml
@@ -35,6 +35,20 @@ jobs:
           command: check
           args: --all-targets --all-features
 
+  msrv:
+    name: Check MSRV
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          submodules: 'true'
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: 1.77
+          override: true
+      - name: Build with MSRV
+        run: cargo build
+
   test:
     name: Run tests
     runs-on: ubuntu-latest

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -11,6 +11,7 @@ Implementation of RFC 8941 and RFC 9651."""
 repository = "https://github.com/undef1nd/sfv"
 keywords = ["http-header", "structured-header", ]
 exclude = ["tests/**", ".github/*"]
+rust-version = "1.77"
 
 [dependencies]
 arbitrary = { version = "1.4.1", optional = true, features = ["derive"] }

